src/components/va-alert/va-alert.tsx

```diff
@@ -65,6 +66,19 @@ export class VaAlert {
   }
 
   private handleAlertBodyClick(e: MouseEvent): void {
+    let headlineText = null;
+
+    // This is the happy path, meaning the user isn't using IE11
+    try {
+      const children = this.el.shadowRoot.querySelector('slot').assignedNodes();
+      const headlineNode = children[0];
+      headlineText = headlineNode.textContent;
+    } catch (e) {
+      // This is where we handle the edge case of the user being on IE11
+      const children = this.el.shadowRoot.childNodes;
+      headlineText = children[0].textContent;
```

Comment:
Above, we grab only the slotted element. This ensures the headline text is the text of the element that we _expect_ to be the headline. Here, we're just grabbing the first child of the alert, which may or may not be a headline.

What happens if that headline slot is left empty? Can we maybe do a `this.el.shadowRoot.querySelector('[slot="headline"]');`? If so, would that eliminate the need for a `try / catch` block?

Reply:
I'll see what happens with that

Reply:
I get `null` from `    console.log(this.el.shadowRoot.querySelector('[slot="headline"]'));`. I tried it with the `<h4>` as well as when it was removed.

Reply:
Interesting. I wonder if that's because it like removes the `slot` property when it actually slots it into the shadow DOM. Ah well, it was worth a shot. Thanks for humoring me. :smile: